ts/Dashboards/Components/HighchartsComponent/HighchartsSyncs/HighchartsHighlightSync.ts

```diff
@@ -224,13 +227,18 @@ const syncPair: Sync.SyncPair = {
                 );
             }
 
-            if (chart && chart.series?.length) {
-                const cursor = e.cursor;
-                if (cursor.type === 'position') {
-                    let series;
+            if (chart && chart.series?.length && cursor.type === 'position') {
+                let series: Series | undefined;
+
+                if (highlightOptions.affectedSeriesId) {
+                    series = chart.get(
```

Comment:
If my `affectedSeriesId`  is wrong and we fail to get the correct series, the sync won't work.
I see a few options to solve this: 
a) add a warning if no series is found
b) pick the first available series (as it works by default) 
c) both above?